.github/renovate.json5

```diff
@@ -360,10 +360,21 @@
 // PR group for Kubernetes CSI
       "groupName": "kubernetes CSI",
       "matchPackagePrefixes": [
-        "kubernetes-csi"
+        "kubernetes-csi",
+        "rook",
       ],
       "separateMajorMinor": "false",
       "pinDigests": false
-    }
+    },
+    {
+// PR group for all the operator framework related things
+      "groupName": "operator framework",
+      "matchPackagePrefixes": [
+        "operator-framework",
+        "redhat-openshift-ecosystem",
+      ],
+      "separateMajorMinor": "false",
+      "pinDigests": false
+    },
```

Comment:
😬 did you  run `renovate-config-validator`?
```suggestion
    }
```

Reply:
Ups!

Reply:
You can validate it in ci, see: https://github.com/crossplane/crossplane/blob/master/.github/workflows/renovate.yml#L31

Reply:
Uhmm I'll add another PR adding the validator, agree @phisco  ?

Reply:
@phisco now there's a linter please proceed =D